### PR TITLE
First example using the pretty context builder.

### PR DESCRIPTION
**Pull request by n3wscott**
I am looking into methods to making logging context less verbose. Here is an example for getting feedback.

### CODE REVIEW

File: pkg/logging/log_context_builder.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import (
+	"fmt"
+)
+
+// LogContextBuilder allows building up log lines with context that is important
+// for debugging and tracing. This class helps create log line formatting
+// consistantly. Logging should always be in the form:
+// <Kind> "<Namespace>/<Name>": <message>
+type LogContextBuilder struct {
+	Kind      string
```
Comment (arschles) [inline pkg/logging/log_context_builder.go]: can you make the type of this `Kind`, so that it can't be set to any arbitrary string?

Comment (n3wscott) [reply pkg/logging/log_context_builder.go]: Oh nice idea!

**Comment by n3wscott** [comment]
Logging is a bit misleading. Might wight want to change it to messages or something. The intent is to never log, but to add memory to message formatting within a class or function. Thoughts?

**Comment by n3wscott** [comment]
I am thinking of renaming the usage of `Log` to `Pretty` in this PR.

**Comment by n3wscott** [comment]
This PR will only change controller.go. Based on feedback we will work the helper to other classes as well.

**Pull request merged**